src/crate/client/exceptions.py

```diff
@@ -19,21 +19,19 @@
 # with Crate these terms will supersede the license and you may use the
 # software solely pursuant to the terms of the relevant commercial agreement.
 
-from .compat import StandardError
 
-
-class Error(StandardError):
+class Error(Exception):
 
     def __init__(self, msg=None, error_trace=None):
-        # for py33 compat.
+        # for py3 compat.
```

Comment:
Might make sense to rephrase the comment here - given that we only support py3 it's kinda strange to point out that this is for py3 compatibility.

I guess now we want this to not break clients already using the message attribute?